domains/blankparticle.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "BlankParticle",
+    "email": "hello@blankparticle.me",
+  }
+  "record": {
+    "CNAME": "www.blankparticle.me"
```

You need to use the `URL` type for redirection. You can know more about it on the [docs](https://is-a.dev/docs).
Example:
```json
{
  "record": {
    "URL": "https://my-other-website.com"
  }
}
```

```suggestion
    "URL": "https://www.blankparticle.me"
```